specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2019-03-01-preview/costmanagement.json

```diff
@@ -2618,6 +2663,54 @@
         }
       }
     },
+    "CheckEligibilityDefinition": {
+      "type": "object",
+      "description": "The check eligibility Connector credentials definition",
+      "properties": {
+        "kind": {
+          "type": "string",
+          "description": "Connector kind (eg aws)"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CheckEligibilityProperties",
+          "description": "Connector credentials properties"
+        }
+      },
+      "x-ms-azure-resource": true
```

Comment:
Technically this isn't an Azure resource (POST bodies are not considered resources), so please don't use `x-ms-azure-resource`

Reply:
will fix